specification/network/resource-manager/readme.md

```diff
@@ -31,24 +31,77 @@ openapi-type: arm
 tag: package-preview-2022-06
 ```
 
+### Tag: package-2022-06-preview
 
-### Tag: package-preview-2022-06
+These settings apply only when `--tag=package-2022-06-preview` is specified on the command line.
 
-These settings apply only when `--tag=package-preview-2022-06` is specified on the command line.
-
-```yaml $(tag) == 'package-preview-2022-06'
+```yaml $(tag) == 'package-2022-06-preview'
 input-file:
   - Microsoft.Network/preview/2022-06-01-preview/network.json
-  - Microsoft.Network/preview/2022-06-01-preview/networkManager.json
-  - Microsoft.Network/preview/2022-06-01-preview/networkManagerActiveConfiguration.json
-  - Microsoft.Network/preview/2022-06-01-preview/networkManagerConnection.json
-  - Microsoft.Network/preview/2022-06-01-preview/networkManagerConnectivityConfiguration.json
-  - Microsoft.Network/preview/2022-06-01-preview/networkManagerEffectiveConfiguration.json
-  - Microsoft.Network/preview/2022-06-01-preview/networkManagerGroup.json
-  - Microsoft.Network/preview/2022-06-01-preview/networkManagerScopeConnection.json
-  - Microsoft.Network/preview/2022-06-01-preview/networkManagerSecurityAdminConfiguration.json
-  - Microsoft.Network/preview/2022-06-01-preview/networkManagerSecurityUserConfiguration.json
+  - Microsoft.Network/preview/2022-06-01-preview/networkManagerGroupMembership.json
+  - Microsoft.Network/stable/2022-07-01/applicationGateway.json
```

Comment:
api-version stable 2022-07-01 is already merged change? just update readme?

Reply:
stable 2022-07-01 is current latest stable version for other services in network. We won't make any changes to them, but we need to add them into the new tag 2022-06-01-preview, or it will fail the SDK/PS Tests